lib/data_scripts/update_bangladesh_regions_script.rb

```diff
@@ -0,0 +1,114 @@
+class UpdateBangladeshRegionsScript < DataScript
+  attr_reader :cache
+  attr_reader :logger
+  attr_reader :results
+  DEFAULT_CSV_PATH = Rails.root.join("db", "bd_regions.csv")
+
+  def self.call(*args)
+    new(*args).call
+  end
+
+  def initialize(dry_run: true, csv_path: DEFAULT_CSV_PATH)
+    super(dry_run: dry_run)
+
+    fields = {module: :data_script, class: self.class.to_s}
+    @logger = Rails.logger.child(fields)
+    @results = {created: Hash.new(0), deleted: Hash.new(0), errors: Hash.new(0), dry_run: dry_run?}
+    @csv_path = csv_path
+    @cache = {state: {}, district: {}, block: {}, facility: {}}
+  end
+
+  def call
+    return unless CountryConfig.current_country?("Bangladesh")
+    destroy_empty_facilities
```

Comment:
We want to clear out existing community facilities that were added, because their regions aren't correct and its easier to add them back from the CSV to ensure correctness.